fmriprep/cli/run.py

```diff
@@ -282,6 +288,26 @@ def main():
     if opts.reports_only:
         sys.exit(int(retcode > 0))
 
+    # Sentry tracking
+    if not opts.notrack:
+        try:
+            from raven import Client
```

Comment:
So even though this is the only place raven is imported tracebacks are automatically captured no matter where they happen?

Reply:
If the exception is raised yes. I'm seeing that some are submitted but others are not. It will probably require some effort on integration with nipype